blocks/library/embed/index.js

```diff
@@ -26,7 +27,32 @@ import BlockAlignmentToolbar from '../../block-alignment-toolbar';
 // These embeds do not work in sandboxes
 const HOSTS_NO_PREVIEWS = [ 'facebook.com' ];
 
-function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, keywords = [] } ) {
+const wpEmbedAPI = memoize( ( url ) => wp.apiRequest( { path: `/oembed/1.0/proxy?${ stringify( { url } ) }` } ) );
```

Comment:
Cache the embed API responses, so if things get transformed, or deleted and added again, we're not spamming the API.

Reply:
>Cache the embed API responses, so if things get transformed, or deleted and added again, we're not spamming the API.

Might be valuable to add as an inline code comment?

Reply:
Added!